CHANGELOG.md

```diff
@@ -18,20 +18,20 @@ We'd like to offer a big thanks to the 11 contributors who made this release pos
 #### Changes
 
 - [DataGrid] Fix `ErrorOverlay` not receiving defined input props (#6819) @banoth-ravinder
-- [DataGrid] Fix conflict with `onResize` added to `React.HTMLAttributes` (#6797) @vizv
+- [DataGrid] Fix conflict with the latest version of `@types/react` (#6797) @vizv
 - [DataGrid] Make more `apiRef` methods private (#6700) @cherniavskii
 - [DataGrid] Provide a clear error message when upgrading (#6685) @oliviertassinari
 - [DataGridPremium] Allow to customize the indent of group expansion toggle (#6837) @MBilalShafi
 - [DataGridPremium] Support aggregating data from multiple row fields (#6656) @cherniavskii
-- [DataGridPro] Fix detail panel not working with `getRowSpacing` prop (#6707) @cherniavskii
+- [DataGridPro] Fix detail panel not working with `getRowSpacing` prop (#6707) @gavbrennan
```

Comment:
This PR was actually mine, my comment was referring to #6733

Reply:
Yeah, sorry, spotted that mistake and did update it with [a later commit](https://github.com/mui/mui-x/pull/6894/commits/200bc5cf2cf710cb1b2c80eb416230940d87b5a9). 🙈 🤦